### PR TITLE
removed pagination if no results

### PR DESCRIPTION
**Pull request by molebox**
This PR removes the paginations links if no results are shown. I was unable to get no results as the search broke, so i couldnt take a screenshot. But it works, trust me :-)

## Pages/Interfaces that will change
Businesses

## Steps to test

Run locally and try and get no results

### CODE REVIEW

File: .env.development.EXAMPLE [hunk 1]
```diff
@@ -1,3 +0,0 @@
-GATSBY_ALGOLIA_APPLICATION_ID=""
-GATSBY_ALGOLIA_SEARCH_KEY=""
-GATSBY_ALGOLIA_INDEX="businesses"
```
Comment (magnificode) [inline .env.development.EXAMPLE:3]: @molebox looks like this may have been accidentally removed!

Comment (molebox) [reply .env.development.EXAMPLE:3]: oh sorry, i just renamed it. Can add back

**Comment by juanri0s** [comment]
[On Business Page we need to hide pagination when no business are matched in search.](https://trello.com/c/WDpG3UVE/202-on-business-page-we-need-to-hide-pagination-when-no-business-are-matched-in-search)

**Comment by ChrisBrownie55** [comment]
Looks like this fixes #211. Closing now.

**Pull request merged**